drivers/SmartThings/zigbee-window-treatment/src/aqara/aqara_utils.lua

```diff
@@ -41,6 +49,7 @@ local function emit_shade_event_by_state(device, value)
   -- update state ui
   if state == SHADE_STATE_STOP or state == 0x04 then
     -- read shade position to update the UI
+    enable_update_shade_level(device)
```

Comment:
What happens if the final level event arrives _before_ the device reports it has stopped? Will that level report never be sent?

Reply:
The report is ignored. However, when the final level event arrives, it requests the last level again with the READ command.

https://github.com/seojune79/SmartThingsEdgeDrivers/blob/3b6c1109489b8b887f39ac3ab17e50ef1f3b0959/drivers/SmartThings/zigbee-window-treatment/src/aqara/aqara_utils.lua#L51-L53